src/box/lua/misc.cc

```diff
@@ -61,6 +61,30 @@ lbox_encode_tuple_on_gc(lua_State *L, int idx, size_t *p_len)
 	return (char *) region_join_xc(gc, *p_len);
 }
 
+/** Convert packed position, tuple or table from Lua to C packed position. */
```

Comment:
Again, mention that the position may be allocated on the fiber region so that whoever uses this code keeps in mind that the region should be cleaned up afterwards. Please mention it **everywehre** a position is returned.